controller/scan/registry.go

```diff
@@ -1507,6 +1507,9 @@ func (rs *Registry) scheduleScanImages(
 				Signature:     false,
 			}
 			sum, ok := rs.summary[info.ID]
+
+			isSignedImage := info.SignatureDigest != "" || (ok && sum.SignatureDigest != "")
```

Comment:
does sum.SignatureDigest really matter for var isSignedImage ?

Reply:
After reviewing the logic, it was not necessary. I've removed it.